app/org/sagebionetworks/bridge/validators/UploadSchemaValidator.java

```diff
@@ -89,9 +89,7 @@ public void validate(Object target, Errors errors) {
 
             // fieldDefinitions
             List<UploadFieldDefinition> fieldDefList = uploadSchema.getFieldDefinitions();
-            if (fieldDefList.isEmpty()) {
-                errors.rejectValue("fieldDefinitions", "requires at least one definition");
-            } else {
+            if (!fieldDefList.isEmpty()) {
                 UploadFieldDefinitionListValidator.INSTANCE.validate(fieldDefList, errors, "fieldDefinitions");
             }
```

Comment:
Is this changing the behavior so that you can save a schema with no fields? If so I approve! It seems like such a schema might just save the rawData column, which might be all researchers want to start.